v7/hierarchical_pages/README.md

```diff
@@ -1,46 +1,46 @@
 This plugin allows to translate paths by specifying paths in a hierarchy.
 
 Assume you have the following hierarchy of posts (default language English):
```

Comment:
Done. Markdown’s spec issues are sometimes annoying. Hopefully, a sensible GFM-cmark library for Python will be out one day.